### PR TITLE
Contact bio

### PR DESCRIPTION
**Pull request by femmebot**
Perview: https://federalist-proxy.app.cloud.gov/preview/18f/nsf-sbir/contact-bio/contact/

Changes proposed in this pull request:
- lightweight style on Bios page
- linked PDs listed on Contact page to their respective bios on Bio page. Program directors' names now point to their bios.
- added bio info to PD YAML file and created a new exec data file 
- merged the updated Contact page layout that Kelly preferred with the PDs at the top and the OIG and abuse contact info at the bottom (see #484)
- converted md content to liquid 

Bonus: added icons to indicate whether links invoke email app or file downloads.

/cc @OpenGlobe @line47 @gemfarmer

### CODE REVIEW

File: pages/contact.md [hunk 1]
```diff
@@ -31,10 +23,25 @@ Notice something wrong with the site or want to make a recommendation? Open an i
...
 </div>
 </section>
+
+
+<section class="usa-section background-white">
+<div class="usa-content utility-content usa-grid">
+<div class="usa-width-one-whole" markdown="1">
+
+## Award fraud or abuse
+If you’re concerned about fraud or abuse with any award, contact the NSF Office of the Inspector General at 703-292-7100 (business hours) or 703-244-4443 (to speak to the duty officer).
```
Comment (gemfarmer) [inline pages/contact.md]: A small thing, but we might consider turning these into phone `<a>`:
```
[703-292-7100](tel:7032927100)
```
and
```
[800-428-2189](tel:8004282189)
```

Comment (gemfarmer) [inline pages/contact.md]: It also might be worth wrapping all of this in [an `<address>` tag](https://www.w3schools.com/tags/tag_address.asp)

File: _includes/executives.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    {% endif %}
+    {% if exec.bio %}
+      <p class="bio">
+        <strong>{{ exec.name }}</strong> {{ exec.bio }}
+      </p>
+    {% endif %}
+
+  </li>
+{% endfor %}
+</ul>
```
Comment (line47) [inline _includes/executives.html]: I'm wondering if there is a way to combine these `executive` and `pd-bio` include files into one as they are almost identical. @gemfarmer do you have any ideas?

Comment (gemfarmer) [inline _includes/executives.html]: @line47 We could combine the two lists before looping over them? Will they be combined though? I'm a bit confused about that...

**Pull request merged**